lib/util/clone.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { clone } from './clone';
+
+describe('util/clone', () => {
+  const obj: any = {
+    name: 'object',
+    type: 'object',
+    isObject: true,
+  };
+
+  it('maintains same order', () => {
+    const res = clone(obj);
+    expect(Object.keys(res)).toEqual(Object.keys(obj));
+  });
+
+  it('assigns "[Circular]" to circular references', () => {
+    obj.circular = obj;
+    const res = clone(obj);
+    expect(res.circular).toBe('[Circular]');
```

Comment:
I expected this to be `null`. Is it the same with fast-safe-stringify?

Reply:
Yes

Reply:
https://www.npmjs.com/package/fast-safe-stringify#Usage